### PR TITLE
Remove lager_transform when running eqc

### PR DESCRIPTION
**Pull request by tolbrino**
This allows the mocking of calls to `lager`.

### CODE REVIEW

**Comment by hanssv** [comment]
Fancy things with rebar3... On a Friday... 🤕

**Comment by tolbrino** [comment]
> Fancy things with rebar3... On a Friday... 🤕

Its all for the greater good of using eqc :-)

File: rebar.config [hunk 1]
```diff
@@ -229,13 +230,16 @@
...
-                                    ]}]}
+            {eqc, [ {project_app_dirs, ["eqc_test"]}
+                  , {extra_src_dirs, [ "eqc/aecore_eqc"
+                                     , "eqc/aega_eqc"
+                                     , "eqc/aeminer_eqc"
+                                     , "eqc/aechannel_eqc"
+                                     , "eqc/aesophia_eqc"
+                                     , "eqc/aeutils_eqc"
+                                     ]}
+                  , {overrides, [{override, [{erl_opts, [debug_info]}]}]}
```
Comment (ThomasArts) [inline rebar.config]: But this means that all other QuickCheck tests that do not use lager mocking are failing... that will give me a lot of emails from CI running the QuickCheck task every few hours...

Comment (tolbrino) [reply rebar.config]: That is correct. I didn't notice that. So we have to configure 2 profiles. (1) `eqc` which works as before; (2) `eqc_lager_mock` which mocks lager.

**Comment by tolbrino** [comment]
Tagging this as wontfix, since it seems obsolete by now.

**Pull request closed without merging**